packages/webview/src/MainContextAware.svelte

```diff
@@ -17,6 +18,7 @@ let initialized = $state(false);
 // Sets the value in the global svelte context
 setContext(States, context.states);
 setContext(Remote, context.remote);
+setContext(Container, context.bindingContainer);
```

Comment:
I'm unsure about giving the Container object directly

we give objects though Context which is native to svelte, while if we give Container object, it's like giving an internal object


and then from a svelte objects it would mean that we're using two ways of retrieving dependencies

Reply:
If we agree on this, I can make some utility methods to simplify getting the instance from a component (and for the tests), so the container is hidden from the component. @benoitf WDYT?